specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeUsages.json

```diff
@@ -0,0 +1,159 @@
+{
+  "parameters": {
+    "resourceUrl": "management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=13&length=10)

In addition to `properties`, you should have `id`, `name`, and `type` properties in all resource responses (even for proxy resources)

Reply:
It based on the Compute usages API - https://docs.microsoft.com/en-us/rest/api/compute/usage/list
Which has same format.
"value": [
    {
      "limit": 2500,
      "unit": "Count",
      "currentValue": 0,
      "name": {
        "value": "availabilitySets",
        "localizedValue": "Availability Sets"
      }
    },

Reply:
The Quota RP can generate Id as resource URI based on following logic- /subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{locationId}/{name.value}
Different RPs have different ways to identify resources, such as batch RP  has  - /subscriptions/{subscriptionId}/resourceGroups/{RGName}/providers/Microsoft.Batch/batchAccounts/{batchAccountName}/{name.value}. 

As per ARM guideline the resource URI should be name value pair, so for  {name.value} should be - /serviceLimits/{name.value}. Does it look fine, then we can follow that convention. This URI doesn't represent actual azure resource URI. - Please suggest?
If the RPs can provide the Ids, then it would have been better option, 
We are  automating the RP onboarding to Quota RP, no special code or logic should be required to onboard any RP, If the RP provides id in their usages response, we can include it.